semapv-properties.tsv

```diff
@@ -0,0 +1,14 @@
+IRI	skos:prefLabel	skos:definition	dc:source	skos:example	rdfs:comment	altLabel	Parent	rdfs:isDefinedBy
+ID	A skos:prefLabel	A skos:definition	A dc:source	A skos:example	A rdfs:comment	A skos:altLabel	SC % SPLIT=|	A rdfs:isDefinedBy
```

Comment:
```suggestion
ID	A skos:prefLabel	A skos:definition	A dc:source	A skos:example	A rdfs:comment	A skos:altLabel	SP % SPLIT=|	A rdfs:isDefinedBy
```

Reply:
Doh! Of course, sorry about that.

Reply:
@matentzn One last clarification needed: For the above to work, the template needs a `TYPE` column, otherwise the terms are still created as `owl:Class` (and have no parents because no SC).

Which raises the question: should these properties by defined as `owl:AnnotationProperty` or as `owl:ObjectProperty`? I think they should be AP, but unless I missed something you never explicitly described them as such in your proposal (#3). The [discussion](https://github.com/information-artifact-ontology/ontology-metadata/issues/90) about the rationale for using OP rather than AP does _not_ make things any clearer to me — rather to the contrary…

Reply:
TYPE must be annotation property - else it does not work with the OWL object model (in which triples of the form <a> <b> <c> are interpreted as annotation assertions most of the time). 

Longer discussion!